nifi-nar-bundles/nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-frontend/src/main/nifi/src/styles.scss

```diff
@@ -293,6 +293,15 @@ a:hover {
     min-width: 760px;
 }
 
+.full-screen-dialog {
+    position: absolute !important;
+    top: 25px;
+    bottom: 25px;
+    right: 25px;
+    left: 25px;
+    max-width: none !important;
+}
+
```

Comment:
You're correct that is not used right now. I was originally going to put the service listing with the process group configuration. However, that is being moved into its own view so this is currently unused. I thought about removing it but left it in place because I suspected we'd need it for other features in the future (docs, custom ui, etc). We are also not using all the other types of dialogs currently.

That said, I'm happy to remove this and we can reintroduce it once it's needed.